pkg/client/add_events.go

```diff
@@ -227,28 +229,18 @@ func (client *DataSetClient) Shutdown() error {
 		RandomizationFactor: client.Config.BufferSettings.RetryRandomizationFactor,
 		Multiplier:          client.Config.BufferSettings.RetryMultiplier,
 		MaxInterval:         client.Config.BufferSettings.RetryMaxInterval,
```

Comment:
This should also be something like - `MaxInterval = min(client.Config.BufferSettings.RetryMaxInterval,  client.Config.BufferSettings.RetryShutdownTimeout / 2)`